lib/client/api.go

```diff
@@ -3438,6 +3438,13 @@ func (tc *TeleportClient) getSSHLoginFunc(pr *webclient.PingResponse) (SSHLoginF
 		}
 		return tc.pwdlessLogin, nil
 	case authType == constants.Local:
+		if pr.Auth.AllowPasswordless && tc.checkIfPwdlessRegisteredWithoutInteraction(pr) {
```

Comment:
@codingllama is is enough to check `AllowPasswordless` flag from server settings or we should check something else? I have checked it but want to double check with you.